scoop/src/main/java/com/lyft/scoop/Router.java

```diff
@@ -32,6 +33,18 @@ public boolean goBack() {
         return false;
     }
 
+    public boolean clear() {
```

Comment:
@itspbj I don't want to introduce extra method. May be router could have minDepth param. So AppRouter have minDepth 1 and DialogRouter has 0.

In this case simple goBack could be still used.